float32 pointX
float32 pointY
---
string response